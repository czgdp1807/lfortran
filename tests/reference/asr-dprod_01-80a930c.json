{
    "basename": "asr-dprod_01-80a930c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/dprod_01.f90",
    "infile_hash": "d1ce33c89bee90c6db80924ecb910e367598d14758aafaa9303aeeb1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-dprod_01-80a930c.stderr",
    "stderr_hash": "a45e6b230aec1aea84144d6576b349447ec7a0edbfda5a5fc097cef0",
    "returncode": 2
}