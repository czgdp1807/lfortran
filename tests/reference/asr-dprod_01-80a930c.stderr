semantic error: Arguments to dprod must be real(4)
 --> tests/errors/dprod_01.f90:2:13
  |
2 |     print*, dprod(4.23_8, 4.3_8)
  |             ^^^^^^^^^^^^^^^^^^^^ 
